### ISSUE TITLE
Int4 inference

### ISSUE THREAD

**Issue by dmenig** [rhymes-ai/Aria#45]
Is there any plan to release an int4 version ? Specifically, I'm interested in the video understanding part.

**Comment by xffxff** [comment]
Yes! You can check out https://github.com/mobiusml/hqq/blob/master/examples/hf/aria_multimodal.py, a HQQ 4-bit version. This implementation is about 4-6x faster and takes 3x less VRAM!

**Comment by AIWarper** [comment]
> Yes! You can check out https://github.com/mobiusml/hqq/blob/master/examples/hf/aria_multimodal.py, a HQQ 4-bit version. This implementation is about 4-6x faster and takes 3x less VRAM!

I was looking into this as I need a local Video Vision solution - is it correct to assume the model was 12 shards at ~4gb a shard?

**Comment by h123fire** [comment]
Error: accessing tensor output of CUDAGraphs that has been overwritten by a subsequent run. Stack trace,/transformers/models/llama/modeling_llama.py", line 1190, in forward
